var/spack/repos/builtin/packages/trilinos/package.py

```diff
@@ -260,7 +260,7 @@ class Trilinos(CMakePackage, CudaPackage, ROCmPackage):
         msg='Cannot build Trilinos with STK as a shared library on Darwin.'
     )
     conflicts('+adios2', when='@:12.14.1')
-    conflicts('cxxstd=11', when='@master:')
+    conflicts('cxxstd=11', when='@13:')
```

Comment:
```suggestion
    conflicts('cxxstd=11', when='@13.2:')
```
Sorry I didn't notice this earlier, but 13.0.1 still allows C++11 (and in fact our code's current setup relies on this because we're still working on upgrading our outdated compiler toolchain).